roles/pd/vars/default.yml

```diff
@@ -53,8 +53,14 @@ log:
     #log-rotate: true
 
 metric:
+  # prometheus client push interval, set "0s" to disable prometheus.
+  interval: "15s"
+  # prometheus pushgateway address, leaves it empty will disable prometheus.
+  address: ""
 
 schedule:
+  max-merge-region-size: 20
+  max-merge-region-keys: 200000
```

Comment:
I think we were intentional to turn merge off in ansible

Reply:
Merge off now or later? :smile:

Reply:
Region merge is turned off without these 2 lines.

Reply:
@disksing  PTAL